src/Symfony/Component/Console/Command/Command.php

```diff
@@ -32,6 +32,7 @@ class Command
     public const SUCCESS = 0;
     public const FAILURE = 1;
     public const INVALID = 2;
+    public const ABORTED = 3;
```

Comment:
Should it be `130`? https://tldp.org/LDP/abs/html/exitcodes.html

Reply:
Hi @chalasr 

I just add a use case in my PR description.
Here aborted means that the user has decided himself to cancel the command.

Reply:
@fabpot  maybe we should call this `CANCELED` instead of `ABORTED` ?